File: AbstractFactory/LogisticsExample/AbstractFactory/Products/Transports/Ship.cs
using AbstractFactory.interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AbstractFactory.products.transports
{
    public class Ship : ITransport
    {
        public double Maxkg => 100000;

        public string Deliver()
        {
            return "Delivering by ship.";
        }
    
    }
}

File: FactoryPattern/Java/src/products/Ship.java
package products;
import interfaces.ITransport;

public class Ship implements ITransport {
    @Override
    public double getMaxCargoKg() {
        return 500000;
    }

    @Override
    public String deliver() {
        return "Доставка с КОРАБ по море";
    }
}
